MovieSharing/ImageObject.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  ImageObject.swift
+//  MovieSharing
+//
+//  Created by Mariam Korashi on 01.10.21.
+//
+
+import Foundation
+
+struct ImageObject: Decodable {
```

Comment:
- [ ] consider renaming to `ThumbnailData` or `ImageData`, as `Object` is implied when you create single instances of a struct and therefore kind of redundant